packages/atlas/src/components/_nft/NftWidget/NftHistory.tsx

```diff
@@ -0,0 +1,178 @@
+  const [isOpen, toggleIsOpen] = useToggle(true)
+  const { convertToUSD } = useTokenPrice()
+
+  const dummyData: HistoryItemProps[] = [
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Bid made by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Listed by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      copy: 'Auction settled by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Auction won by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Bid made by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Listed by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      copy: 'Auction settled by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Auction won by',
+      memberHandle: 'Madness',
+    },
+  ]
+
+  return (
+    <>
+      <NftHistoryHeader data-open={isOpen} data-size={size} onClick={toggleIsOpen}>
+        <Text variant={size === 'small' ? 'h300' : 'h400'}>History</Text>
+        <StyledChevronButton data-open={isOpen} iconOnly variant="tertiary" icon={<SvgActionChevronB />} />
+      </NftHistoryHeader>
+      {isOpen && (
+        <div style={{ position: 'relative' }}>
+          <FadingBlock data-size={size} width={width} />
+          <HistoryPanel width={width} data-size={size} data-open={isOpen}>
+            {/* TODO: remove dummy data */}
+            {dummyData.map((props: HistoryItemProps, index) => (
+              <HistoryItem key={index} {...props} />
+            ))}
+          </HistoryPanel>
+          <FadingBlock data-size={size} width={width} data-bottom />
+        </div>
+      )}
+    </>
+  )
+}
+
+type HistoryItemProps = {
+  size: Size
+  memberAvatarUri: string
+  date: Date
+  joyAmount?: number
+  dollarValue?: string
+  copy: string
+  memberHandle: string
+}
+export const HistoryItem: React.FC<HistoryItemProps> = ({
```

Comment:
I think we could move this component to separate file, this one is big enough. WDYT?

Reply:
I think is fine like this, once the dummy data is removed the file is going to be fairly small anyways